lib/routes/account.js

```diff
@@ -1021,50 +1045,78 @@ module.exports = function (
         validate: {
           payload: {
             uid: isA.string().max(32).regex(HEX_STRING).required(),
-            code: isA.string().min(32).max(32).regex(HEX_STRING).required()
+            code: isA.string().min(32).max(64).regex(HEX_STRING).required()
```

Comment:
> Why do the token-verification codes need to be bigger than the email-verification codes?

I think initially it was to increase entropy because we weren't verifying the `uid` when verifying the `tokenVerificationId`. But then I did the paranoid implementation, so that's moot. Happy to make it more uniform in the db PR if people like?